src/utils/beacon/beacon.tsx

```diff
@@ -30,6 +31,30 @@ export const useRemovePeer = () => {
       .then(refresh);
 };
 
+export const useRemovePeersByAccounts = () => {
+  const client = useQueryClient();
+  const getPeersForAccounts = useGetPeersForAccounts();
+
+  return async (pkhs: RawPkh[]) => {
+    const peersToRemove = getPeersForAccounts(pkhs);
+    const peersData = await client.fetchQuery({
+      queryKey: [PEERS_QUERY_KEY],
+      // getPeers actually returns ExtendedPeerInfo (with the senderId)
+      queryFn: () => WalletClient.getPeers() as Promise<ExtendedPeerInfo[]>,
+    });
+
+    await Promise.all(
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
why?

Reply:
I was confused by this bit of code https://github.com/trilitech/umami-v2/blob/34249f6fbf62fe071ddc2f45d013c415b3f908e7/src/utils/beacon/BeaconPeers.tsx#L36

You right, it seems to be unnecessary here

Reply:
Fixed now